sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -990,27 +1008,23 @@ ur_program_handle_t ProgramManager::getBuiltURProgram(
     // emplace all subsets of the current set of devices into the cache.
     // Set of all devices is not included in the loop as it was already added
```

Comment:
I think this comment ("Set of all devices is not included...") needs to be moved to the line 832, if I understand the logic correctly.

Reply:
fixed https://github.com/intel/llvm/pull/16334/commits/19983fe9bd65e3f9008cd3e244886c89dd547a3d